packages/kuma-gui/src/app/application/components/route-view/RouteView.vue

```diff
@@ -89,6 +91,11 @@ const win = window
 const env = useEnv()
 const can = useCan()
 const uri = useUri()
+
+
+let resolve: (resolved: object) => void
+const meResponse = new Promise((r) => resolve = r)
```

Comment:
Boilerplate for setting up the JS accessible promise to wait for `me` to resolve.